Makefile

```diff
@@ -90,3 +92,9 @@ install: setup.data rbac_static.csv
 	scripts/install.sh 755 block_device_io.native $(DESTDIR)$(LIBEXECDIR)/block_device_io
 # Libraries
 	ocaml setup.ml -install
+# xenserver html docs
+	mkdir -p $(DESTDIR)$(DOCDIR)/html/xenserver
+	cp -r -L _build/ocaml/doc/api $(DESTDIR)$(DOCDIR)/html/xenserver
+	cp ocaml/doc/{*.js,*.html,*.css,*.png} $(DESTDIR)$(DOCDIR)/html/xenserver
```

Comment:
This is potentially not portable because Make is not guaranteed to use Bash and this is a Bash glob. Make is using the shell from SHELL (if it is set) and `/bin/sh`) otherwise. Here is a portable way:

```
cd ocaml/doc && cp *.js *.html *.css *.png $(DESTDIR)$(DOCDIR)/html/xenserver
```